mewarpx/mewarpx/diags_store/checkpoint_diagnostic.py

```diff
@@ -23,16 +23,15 @@ def __init__(self, diag_steps, name=init_restart_util.default_checkpoint_name,
                 Also plot on this period if enabled.
             name (str): The name of the diagnostic to be passed into the
                 picmi checkpoint diagnostic.
-            write_dir (str): The directory where diagnostic data will be written
-                from the checkpoint diagnostic.
-            kwargs: For a list of valid keyword arguments see diag_base.WarpXDiagnostic
+            kwargs: For a list of valid keyword arguments see
+                diag_base.WarpXDiagnostic
         """
         self.diag_steps = diag_steps
         self.name = name
-        self.write_dir = write_dir
 
         super(CheckPointDiagnostic, self).__init__(diag_steps, **kwargs)
 
+        self.write_dir = self.DIAG_DIR
```

Comment:
Since all the diags will have the same `DIAG_DIR`, does it make sense to make one global variable for `DIAG_DIR` that all the diags will use. We can put it in `diag_base` or something.

Reply:
That is how it is done, in `diag_base` we declare the class variable `DIAG_DIR = 'diags'` and then all diagnostics references it. Setting the `self.write_dir` in each diagnostic accounts for the specific folder of that diagnostic type (see 'field_diagnostic.py' line 59 for example). The checkpoint diagnostic just happens to not have a specific sub-directory (although we could create one for it as well).